install/kubernetes/cilium/values.yaml

```diff
@@ -1581,3 +1581,12 @@ clustermesh:
 externalWorkloads:
   # -- Enable support for external workloads, such as VMs (false by default).
   enabled: false
+
+# -- Configure BGP
+bgp:
```

Comment:
I put a lot of effort into alphabetizing values, please keep that work in place.

Reply:
@seanmwinn No alphabetization can resist a horde of developers unless it's tested in CI :wink: